app/views/spree/admin/shared/_tabs.html.haml

```diff
@@ -1,6 +1,6 @@
 = tab :dashboard, :route => :admin, :icon => 'icon-dashboard'
 = tab :products , :option_types, :properties, :prototypes, :variants, :product_properties, :taxons, :url => admin_products_path, :icon => 'icon-th-large'
-= tab :order_cycles, :url => main_app.admin_order_cycles_path
+= tab :order_cycles, :url => main_app.admin_order_cycles_path, :icon => 'icon-th-large'
```

Comment:
I found this class in `spree-46d6f8f5fd43/backend/app/assets/stylesheets/admin/plugins/_font-awesome.scss`. Maybe `.icon-repeat` or `.icon-refresh` would work better?

Reply:
awesome Maikel! I switched to icon-refresh:
![Screenshot 2019-07-26 at 10 52 47](https://user-images.githubusercontent.com/1640378/61943563-9ff1fb80-af93-11e9-8bc4-fd00884f6761.png)

It's not like amazingly appropriate, but I think it's better :-D

Reply:
@sauloperez @RachL ☝️ ok?

Reply:
Better, yes!